src/main/resources/META-INF/rewrite/quarkus-upgrade.yml

```diff
@@ -15,14 +15,23 @@
 #
 ---
 type: specs.openrewrite.org/v1beta/recipe
-name: org.openrewrite.java.quarkus.Quarkus1_11to1_12Migration
-displayName: Uprgade Quarkus project from 1.11 to 1.12
-description: Uprgade Quarkus project from 1.11 to 1.12.
+name: org.openrewrite.java.quarkus.Quarkus1_11to1_13Migration
+displayName: Quarkus 1.13 migration from Quarkus 1.11
+description: Migrates Quarkus 1.11 to 1.13.
```

Comment:
Consider naming as per `rewrite-spring` to just be `Quarkus1_13Migration` -- but, not so important as to actually warrant changes. Just for consideration? https://github.com/openrewrite/rewrite-spring/blob/4b4677e612988e5518e9f5d2cc9d5e8c0a4f85fa/src/main/resources/META-INF/rewrite/spring-boot-configuration-migration.yml#L1031-L1033